owlplug-client/src/main/java/com/owlplug/core/components/ApplicationDefaults.java

```diff
@@ -77,6 +77,7 @@
   public static final String SELECTED_ACCOUNT_KEY = "SELECTED_ACCOUNT_KEY";
   public static final String SYNC_PLUGINS_STARTUP_KEY = "SYNC_PLUGINS_STARTUP_KEY";
   public static final String STORE_DIRECTORY_ENABLED_KEY = "STORE_DIRECTORY_ENABLED_KEY";
+  public static final String STORE_BY_MAKER_ENABLED_KEY = "STORE_BY_MAKER_ENABLED_KEY";
```

Comment:
Maybe a key definition like `STORE_CREATOR_DIRECTORY_ENABLED_KEY` will be more accurate.
* **Store** means the Store (~merchant feature) rather than storage action
* I think we should use the word `creator` as it's the term used on classes an models.

Reply:
I agree I will make this change! I mad this key before I discovered the other class in the codebase 😅